### DISCUSSION TITLE
3.8.0

### DISCUSSION THREAD

**Discussion by gsemet** [Guake/guake#1927] [Announcements]
This has been a long time since the latest official releases. More than one year without real update, unfortunately. I have pretty much stepped back on the role of maintainer, linux isn't my main development environment anymore, and Guake is so linked with the GTK system I cannot debug or do anything without having to start a VM.

@Davidy22 and @mlouielu have started taking care of Guake and they are helping a lot on the boring duties of the package maintainance: ticket triage, bugfixes, preparing releases...

There have been lot of minor changes and bug fixes it is not easy to summaries in one sentence. See the [release page for more information](https://github.com/Guake/guake/releases/tag/3.8.0).

**Comment by Davidy22** [comment]
Went over to pypi to check the package and it seems like it's still on 3.7. Tried attaching a wheel to this comment,  but github apparently doesn't like that. If you can't build the wheel on a mac I can try find some file sharing site to send one built on my machine

**Comment by Davidy22** [comment]
Guake pypi is still on 3.7. Is there trouble with accessing pypi? In other external site matters, has checking readthedocs turned anything up? If we've lost pypi credentials I think there's an alt channel to get the name at least, don't know about readthedocs.

**Comment by Davidy22** [comment]
I'm going to submit a request to get ownership of the guake pypi and readthedocs page transferred if you're having trouble accessing your account for them @gsemet. I'll wait a few days first in case you can get back with a reply before then.

**Reply by gsemet** [reply]
i do not have any issue with guake pipy, works great. BUT the release it not updated because the metadata of the 3.8 package it not good. someone (not me, I do not have a dev envionment) needs to execute the make release target to update the version that is hardcoded in the executable, and then in the wheel.

for the readthedoc, the build fails and so needs to be fixed => https://github.com/Guake/guake/issues/1945

**Reply by Davidy22** [reply]
Alright, I'll do that and send you a download link for the wheel somehow, going to tag a new release as soon as the keyboard shortcut fix finally finishes getting reviewed and merged so I'll make the wheel for next version.

Looking at the dump in linked issue and there's something ungodly happening here,  but I'll see what I can figure out.

**Reply by Davidy22** [reply]
Tagged 3.8.1, so I'll prepare a wheel in a bit.
